src/widgets/_windows.scss

```diff
@@ -117,6 +117,23 @@ window:not(.popup):not(.menu) {
             text-shadow: none;
             -gtk-icon-shadow: none;
         }
+
+        pane-separator {
+            background-color: rgba(black, 0.0);
+            box-shadow: none;
+        }
+
+        separator {
+            background: none;
+            margin: 0 rem(18px);
+        }
+
+        separator:first-child,
+        separator:last-child {
+            margin: 0;
+            min-width: 0;
+            padding: 0;
```

Comment:
Is there a reason to believe that separators have any of these properties set by default? So far it looks like the only rules we have added for separators have been nested inside other elements

Reply:
With these removed, the separator is thin like it was in the previous stylesheet.